upload.go

```diff
@@ -127,7 +909,7 @@ func processUpload(upReq UploadRequest) (upload Upload, err error) {
 	}
 
 	if len(extension) == 0 {
-		extension = "ext"
+		extension = extensions[mimetype]
```

Comment:
Exactly that, it was meant to be generic vs "txt" being for text files. If you think there's something even better, I'm all ears.  I had considered .bin at a time but that implies the file is binary.